bolt-design-system/bolt
# Fix/ie edge final fix

## Jira

https://app.asana.com/0/1126340469288208/1138725139228457/f

## Summary

Resolve IE/Edge styling differences

## Details

This resolves all IE/Edges styling differences minus the Dropdown. The only outstanding task on IE/Edge was horizontal overflow, and that is resolved here in this PR.

## How to test

Check out Micro Journeys, compare IE/Edge to Chrome or FF. If there are new issues, create a new ticket and assign to @glassdimly.
